app/assets/v2/js/pages/profile-trust.js

```diff
@@ -648,6 +663,113 @@ Vue.component('brightid-verify-modal', {
   }
 });
 
+Vue.component('duniter-verify-modal', {
+  delimiters: [ '[[', ']]' ],
+  data: function() {
+    return {
+      showValidation: false,
```

Comment:
We're providing `showValidation` as a prop and defining a default value there - we don't need this ln 👍